src/commands/instance/list.ts

```diff
@@ -0,0 +1,79 @@
+import execa from 'execa'
+
+import { Kommand } from '../../common'
+
+export class InstanceList extends Kommand {
+  static initSdk = false
+
+  static description = 'Lists the Kuzzle running instances'
+
+  async runSafe() {
+    const instancesList = await this.getInstancesList()
+    // Disable eslint no-console to allow using console.table
+    // Instead of add another dependency to kourou
```

Comment:
Console.table is self explanatory :100: 
```suggestion
    // Disable eslint no-console to allow using console.table
    // Instead of add another dependency to kourou
```